### PR TITLE
remove minishift support from makefiles

### PR DESCRIPTION
**Pull request by ranakan19**
fix wrt: https://issues.redhat.com/browse/CRT-904

### CODE REVIEW

File: make/test.mk [hunk 1]
```diff
@@ -96,16 +95,10 @@ print-operator-logs:
...
 login-as-admin:
 ifeq ($(OPENSHIFT_BUILD_NAMESPACE),)
-    ifeq ($(IS_CRC),)
-        ifneq ($(IS_OS_3),)
-        	# is running locally and against OS 3, so we assume that it's minishift
-			$(info logging as system:admin")
-			oc login -u system:admin 1>/dev/null
-        endif
-    else
+    ifneq ($(IS_CRC),)
```
Comment (alexeykazakov) [inline make/test.mk:98]: I would just drop `login-as-admin` completely. We expect the user to be logged into cluster as an admin no matter what cluster it is. No special login treatment for CRC. Just to simplify the flow.

File: make/test.mk [hunk 2]
```diff
@@ -355,9 +323,8 @@ deploy-operator:
 	$(eval COMPONENT_IMAGE_REPLACEMENT := ;s|REPLACE_REGISTRATION_SERVICE_IMAGE|${REGISTRATION_SERVICE_IMAGE_NAME}|g)
 	$(eval MEMBER_OPERATOR_WEBHOOK_IMAGE := $(shell cat ${IMAGE_NAMES_DIR}/member-operator-webhook))
 	$(eval COMPONENT_IMAGE_REPLACEMENT := ${COMPONENT_IMAGE_REPLACEMENT};s|REPLACE_MEMBER_OPERATOR_WEBHOOK_IMAGE|${MEMBER_OPERATOR_WEBHOOK_IMAGE}|g)
-ifeq ($(IS_OS_3),)
-		# it is not using OS 3 so we will install operator via CSV
-		$(eval NAME_SUFFIX := ${QUAY_NAMESPACE}-${RESOURCES_SUFFIX})
+	# it is not using OS 3 so we will install operator via CSV
```
Comment (alexeykazakov) [inline make/test.mk:326]:
```suggestion
	# install operator via CSV
```

**Review by alexeykazakov — COMMENTED** [review]
Looks good overall!

**Review by alexeykazakov — APPROVED** [review]

File: make/test.mk [hunk 3]
```diff
@@ -15,9 +15,7 @@ TEST_NS := ${QUAY_NAMESPACE}-toolchain-e2e-${DATE_SUFFIX}
 AUTHOR_LINK := $(shell jq -r '.refs[0].pulls[0].author_link' <<< $${CLONEREFS_OPTIONS} | tr -d '[:space:]')
 PULL_SHA := $(shell jq -r '.refs[0].pulls[0].sha' <<< $${CLONEREFS_OPTIONS} | tr -d '[:space:]')
 
-IS_CRC := $(shell oc config view --minify -o jsonpath='{.clusters[0].cluster.server}' 2>&1 | grep crc)
 IS_KUBE_ADMIN := $(shell oc whoami | grep "kube:admin")
```
Comment (MatousJobanek) [inline make/test.mk:18]: this is not needed anymore 
```suggestion
```

File: make/test.mk [hunk 4]
```diff
@@ -305,21 +269,13 @@ ifeq ($(SET_IMAGE_NAME),)
...
     	# check if it is running locally
         ifeq ($(OPENSHIFT_BUILD_NAMESPACE),)
-                # if it is running locally, then build the image via docker
-            ifneq ($(IS_OS_3),)
-            	# is running locally and against OS 3, so we assume that it's minishift - it will use local docker registry
-				$(eval IMAGE_NAME := docker.io/${GO_PACKAGE_ORG_NAME}/${REPO_NAME}:${DATE_SUFFIX})
-				$(MAKE) -C ${E2E_REPO_PATH} build
-				docker build -f ${E2E_REPO_PATH}/build/Dockerfile -t ${IMAGE_NAME} ${E2E_REPO_PATH}
-            else
             	# if is using OS4 then use quay registry
```
Comment (MatousJobanek) [inline make/test.mk:272]:
```suggestion
            	# if is running locally, then build the image and push to quay repository
```

**Review by MatousJobanek — APPROVED** [review]
Nice cleanup :+1:

**Comment by ranakan19** [comment]
closing this PR, created a new PR https://github.com/codeready-toolchain/toolchain-e2e/pull/288/files - with the same changes but different name

**Pull request closed without merging**